libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -304,6 +304,9 @@ static BaseType_t prvSecureConnectHelper( Socket_t xSocket,
 static BaseType_t prvNonSecureConnectHelper( Socket_t xSocket,
                                              SocketsSockaddr_t * pxHostAddress )
 {
+    /* Disable unused parameter warning. */
+    (void) xSocket;
```

Comment:
`( void )` instead of `(void)` across all the changes please.

Reply:
Done